### DISCUSSION TITLE
Arduino Nano

### DISCUSSION THREAD

**Discussion by realcnk** [sblantipodi/firefly_luciferin#196] [Q&A]
Hi, i tried to flash via web the full firmware in Arduino Nano but not works. How i can use luciferin in Arduino Nano? Actualy works with fast led and ambibox.

Note: Aurduino Nano 33 IOT

**Comment by sblantipodi** [comment]
Hi @realcnk ,
thanks for the interest in Luciferin.

Luciferin does not support Arduino Nano and there is no plan in adding that support.
ESP8266 or ESP32 are way cheaper and more powerfull, I suggest an upgrade if you are interested in Luciferin.

**Comment by realcnk** [comment]
I'll upgrade to ESP32 then

Thx for the reply.

**Comment by sblantipodi** [comment]
no problem.
you can find a lot of infos on the [WIKI](https://github.com/sblantipodi/firefly_luciferin/wiki)
most ESP32 and ESP8266 are supported.

**Reply by realcnk** [reply]
I bought an ESP32 board and assembled the whole system and it works much better than my previous system with Arduino and Ambibox. Great job.

**Comment by sblantipodi** [comment]
glad that you liked it, thanks for the feeedback ;)
